src/main/java/org/folio/rest/impl/ServicePointAPI.java

```diff
@@ -156,6 +159,16 @@ public void postServicePoints(String lang, Servicepoint entity,
           Context vertxContext) {
     vertxContext.runOnContext(v -> {
       try {
+
+        String validateSvcptResult = validateServicePoint(entity) ;
+        if (validateSvcptResult != null && !validateSvcptResult.isEmpty()){
+          asyncResultHandler.handle(Future.succeededFuture(
+            PostServicePointsResponse.respond422WithApplicationJson(
+              ValidationHelper.createValidationErrorMessage("name",
+                entity.getName(), validateSvcptResult))));
```

Comment:
Can you move `ValidationHelper.createValidationErrorMessage("name", entity.getName(), validateSvcptResult)` into validateServicePoint?

Reply:
The only gain by doing this is hiding away a few lines of codes for reusing purposes. However, would it be clearer to leave them there to be consistent with the overall pattern in the class?